src/views/calendar.hbs

```diff
@@ -2,5 +2,6 @@
 <title>{{{__ "calendar_title"}}} – {{{__ "common_title"}}}</title>
 {{/section}}
 
-<iframe width="100%" height="100%" frameborder="0" src="https://teamup.com/ksb15466a44b299f51"
-  style="overflow: hidden"></iframe>
+<div class="embed-responsive embed-responsive-4by3" style="height: 100%;">
+  <iframe class="embed-responsive" src="https://teamup.com/ksb15466a44b299f51"></iframe>
```

Comment:
According to [the docs](https://getbootstrap.com/docs/4.0/utilities/embed/), iframe should have `embed-responsive-item` class.

Reply:
Changed it